### ISSUE TITLE
7.3 branch consistency check failing

### ISSUE THREAD

**Issue by ywelsch** [elastic/elasticsearch#47084]
This has been failing since nearly two weeks: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.3+branch-consistency/

```
Execution failed for task ':verifyVersions'.
> java.lang.Exception: .ci/bwcVersions is outdated, run `./gradlew updateCIBwcVersions` and check in the results
```

**Comment by jasontedor** [comment]
> This has been failing since nearly two weeks

Note that we didn't have build triage for a large fraction of that period. You could run the command specified in the output to fix this. I pushed 3c9b9b693394f55f16334c6bf13e3fe93b14a6f8.

**Comment by jasontedor** [comment]
Closed by 3c9b9b693394f55f16334c6bf13e3fe93b14a6f8
